lib/facter/rhsm_repos.rb

```diff
@@ -1,8 +1,9 @@
+require 'facter'
```

Comment:
this require shouldn't be needed, facts should work out of the box. Did you run into an error without it?